Dockerfile

```diff
@@ -4,6 +4,8 @@ ARG TARGETOS
 ARG TARGETARCH
 
 WORKDIR /workspace
+RUN chmod 755 /workspace
```

Comment:
Non-blocking: The go-toolset image defaults the WORKDIR to `/opt/app-root/src`. In the future we can drop the WORKDIR + chmod command, saving us 2 layers in the build.